extensions/pom.xml

```diff
@@ -36,6 +36,7 @@
     <modules>
         <module>jackson</module>
         <module>orgjson</module>
+	<module>gson</module>
```

Comment:
Minor formatting fix request.

Reply:
this is a tab, needs be be 8 spaces instead